static/js/components/GcnEventPage.jsx

```diff
@@ -263,203 +269,210 @@ const GcnEventPage = ({ route }) => {
-          >
-            <Typography className={styles.accordionHeading}>
-              Galaxies within localization
-            </Typography>
-          </AccordionSummary>
-          <AccordionDetails>
-            {gcnEventGalaxies?.sources.length === 0 ? (
-              <Typography variant="h5">None             </Typography>
-            ) : (
+            </AccordionDetails>
+          </Accordion>
+        </div>
+      </Grid>
+      <Grid item xs={5}>
+        <div className={styles.columnItem}>
+          <Accordion defaultExpanded>
+            <AccordionSummary
+              expandIcon={<ExpandMoreIcon />}
+              aria-controls="gcnEvent-content"
+              id="info-header"
+            >
+              <Typography className={styles.accordionHeading}>
+                Event Information
+              </Typography>
+            </AccordionSummary>
+            <AccordionDetails>
               <div className={styles.gcnEventContainer}>
-                             {" "}
-                <GalaxyTable galaxies={gcnEventGalaxies.sources} />           {" "}
+                <Link to={`/gcn_events/${gcnEvent.dateobs}`}>
+                  <Button color="primary">
+                    {dayjs(gcnEvent.dateobs).format("YYMMDD HH:mm:ss")}
+                  </Button>
+                </Link>
+                ({dayjs().to(dayjs.utc(`${gcnEvent.dateobs}Z`))})
               </div>
-            )}
-          </AccordionDetails>
-        </Accordion>
-      </div>
-      <div className={styles.columnItem}>
-        <Accordion defaultExpanded>
-          <AccordionSummary
-            expandIcon={<ExpandMoreIcon />}
-            aria-controls="gcnEvent-content"
-            id="observationplan-header"
-          >
-            <Typography className={styles.accordionHeading}>
-              Observation Plans
-            </Typography>
-          </AccordionSummary>
-          <AccordionDetails>
-            <div className={styles.gcnEventContainer}>
-              <ObservationPlanRequestForm
-                gcnevent={gcnEvent}
-                action="createNew"
-              />
-              <ObservationPlanRequestLists gcnEvent={gcnEvent} />
-            </div>
-          </AccordionDetails>
-        </Accordion>
-      </div>
-    </div>
+            </AccordionDetails>
+          </Accordion>
+        </div>
+        <div className={styles.columnItem}>
+          <Accordion defaultExpanded>
+            <AccordionSummary
+              expandIcon={<ExpandMoreIcon />}
+              aria-controls="gcnEvent-content"
+              id="lightcurve-header"
+            >
+              <Typography className={styles.accordionHeading}>
+                Light curve
+              </Typography>
+            </AccordionSummary>
+            <AccordionDetails>
+              <div className={styles.gcnEventContainer}>
+                {gcnEvent.lightcurve && (
+                  <div>
+                    {" "}
```

Comment:
Do we still need this funky spacing here?

Reply:
Yeah this was a holdover from maybe the first iteration of this page. Deleting.